seneca.js

```diff
@@ -181,7 +181,10 @@ var internals = {
     legacy: {
 
       // use old error codes, until version 3.x
-      error_codes: true
+      error_codes: true,
+
+      // use parambulator for message validation, until version 3.x
+      validate: true
```

Comment:
what does "until version 3" mean? Can we just flag it as "false from v4?" or "remove in v4"?